File: C/lt_01_estseq/lt01_estseq_06.c
#include <stdio.h>

int main() {

  float X, Y, aux;

  printf("Insira o Valor de X: ");
  scanf("%f", &X);
  printf("Insira o Valor de Y: ");
  scanf("%f", &Y);

  aux = X;
  X = Y;
  Y = aux;

  printf("O valor de X e %.2f.\nO Valor de Y e %.2f.\n", X, Y);
  return 0;
}

File: rust/lt01-estseq/lt01-estseq-06.rs
use std::io;

fn main() {
    // read x
    println!("Insira o valor de x:");
    let mut x = String::new();
    io::stdin().read_line(&mut x).expect("Error read_line!");
    // read y
    println!("Insira o valor de y:");
    let mut y = String::new();
    io::stdin().read_line(&mut y).expect("Error read_line!");
    // switch
    let aux = x;
    x = y;
    y = aux;
    println!("x={}y={}", x, y);
}
